### DISCUSSION TITLE
Separate `natural-scroll` settings for workspace scroll and view scroll

### DISCUSSION THREAD

**Discussion by Chiffario** [YaLTeR/niri#402] [Ideas]
Touchpad input could support having separate values for workspace and view scroll directions or more values for such cases, i.e. `natural-scroll vertical-only`. A different solution could be adding a `gestures` configuration field that allows overriding the scroll direction for gestures (along with potential new configuration later on)

Edit: Just saw #372 that kind of supports the latter way of implementing this

Note: This is absurdly minor and I might actually be the only person who wants this, but that's why I am making the discussion :^)

**Comment by YaLTeR** [comment]
That sounds a bit weird indeed, but I guess with gesture bindings, we'll be able to put properties on them for this kind of customization.
